dotCMS/src/main/java/com/dotmarketing/servlets/BinaryExporterServlet.java

```diff
@@ -385,7 +385,9 @@ public void doGet(HttpServletRequest req, HttpServletResponse resp) throws Servl
 
       if(params.get("quality_q")!=null) {
       	final UserAgent userAgent = new UserAgent(req.getHeader("user-agent"));
-        if(userAgent.getBrowser() == Browser.SAFARI || userAgent.getOperatingSystem().getGroup() == OperatingSystem.IOS){
+        if((userAgent.getBrowser() == Browser.SAFARI
+				&& Integer.parseInt(userAgent.getBrowserVersion().getMajorVersion())<14)
+				|| userAgent.getOperatingSystem().getGroup() == OperatingSystem.IOS){

```

Comment:
ios 14 can support webp as well, we need to remove the condition.

https://caniuse.com/?search=webp